kuzzleio/kuzzle-admin-console
# Document blocks list view

~depends on #441~

## What does this PR do ?
This PR introduces the document boxes view in the Document List. 

### How should this be manually tested?
* Open the content of a non-empty collection in the Data section
* Click on the "Blocks" icon on the right of the quick search filter
* You should see the documents in form of blocks
